src/components/20-molecules/policy-features/README.md

```diff
@@ -0,0 +1,109 @@
+# Policy features
+
+With this component you can highlight as much `features` as you want. You can set a main title and put `features` items as childs to it. This component changes sizes according to the viewport.
+
+## Usage
+
+**Important:** If this component needs to run in Internet Explorer 11, [you need to use our polyfill](https://github.com/axa-ch/patterns-library/tree/develop/src/components/05-utils/polyfill).
+
+```bash
+npm install @axa-ch/policy-features
+```
+
+```js
+import '@axa-ch/policy-features';
+```
+
+```html
+<axa-policy-features title="Your main title" axaStyle="dark-indigo">
+  <axa-policy-features-item
+    title="The title of item"
+    icon="email"
+    description="A text which can be as long as you want."
+  ></axa-policy-features-item>
+</axa-policy-features>
+```
+
+### React
+
+You can use this component directly in your JSX without further React-ification (see Usage above).
+
```

Comment:
Why is testimonials a molecule and not an organism?

Reply:
Marco, Luca and me had a discussion about that. For Marco and me this component has not much complexity for an organism.